libbindgen/tests/headers/inline_namespace.hpp

```diff
@@ -0,0 +1,11 @@
+// bindgen-flags: --enable-cxx-namespaces -- -std=c++11
+
+namespace foo {
+  inline namespace bar {
+    using Ty = int;
+  };
```

Comment:
So, that's a good question. Multiple nested inline namespaces work fine with this patch.

Overriding an item defined in an inline namespace is legal, though referencing it is a compile error.

If we want to support this edge case (which I think is kinda pointless tbh), we should keep all the namespaces in Rust I think. That will make bindgen harder to use with the STL though, given you no longer have std::string, but std::__gnu_xxx::string, etc.

This deserves a bit more discussion so I'll move this to another PR.